src/custom/components/Header/index.tsx

```diff
@@ -114,24 +115,29 @@ export default function Header() {
                           handleMobileMenuOnClick()
                         }}
                       >
-                        {darkMode ? (
-                          <>
-                            <SVG src={IMAGE_SUN} description="Sun light mode icon" /> Light
-                          </>
-                        ) : (
-                          <>
-                            <SVG src={IMAGE_MOON} description="Moon dark mode icon" /> Dark
-                          </>
-                        )}{' '}
-                        Mode
+                        <SVG
+                          src={darkMode ? IMAGE_SUN : IMAGE_MOON}
+                          description={`${darkMode ? 'Sun/light mode' : 'Moon/dark'} mode icon`}
```

Comment:
Extra mode:

```suggestion
                          description={`${darkMode ? 'Sun/light' : 'Moon/dark'} mode icon`}
```

Reply:
Sharp eye 👁️ , thanks for the find. Will address in a new upcoming PR.